Encountify/Services/IDataStore.cs

```diff
@@ -3,12 +3,13 @@
 
 namespace Encountify.Services
 {
+    //generic class for every needed place
     public interface IDataStore<T>
     {
-        Task<bool> AddLocationAsync(T location);
-        Task<bool> UpdateLocationAsync(T location);
-        Task<bool> DeleteLocationAsync(int id);
-        Task<T> GetLocationAsync(int id);
-        Task<IEnumerable<T>> GetLocationsAsync(bool forceRefresh = false);
+        Task<bool> AddAsync(T location);
```

Comment:
If IDataStore is now universal, shouldn't the variables also be renamed? Like T location to T item

Reply:
Thanks, totally missed it